test/bounded-memory/mzcompose.py

```diff
@@ -415,17 +415,21 @@ class KafkaScenario(Scenario):
 
             > CREATE INDEX i_accumulable IN CLUSTER idx_cluster ON accumulable(a);
 
+            > SET CLUSTER = idx_cluster;
+
             > SELECT count(*) FROM accumulable;
             10000001
             """
         ),
         post_restart=dedent(
             """
+            > SET CLUSTER = idx_cluster;
+
             > SELECT count(*) FROM accumulable;
             10000001
             """
         ),
-        materialized_memory="4.5Gb",
+        materialized_memory="9.5Gb",
```

Comment:
Just FYI: I had to increase the memory here for this check, which might however be expected since a cluster with size `1-8G` is used. `8.5Gb` was successful locally, but not in CI. `9.5Gb` was successful in CI. Trying out `9.0Gb`.